pkg/synthetictests/operators.go

```diff
@@ -229,6 +230,36 @@ func testOperatorOSUpdateStartedEventRecorded(events monitorapi.Intervals, clien
 	return []*junitapi.JUnitTestCase{success}
 }
 
+// testBackoffPullingRegistryRedhatImage looks for this symptom:
+//   reason/ContainerWait ... Back-off pulling image "registry.redhat.io/openshift4/ose-oauth-proxy:latest"
+//   reason/BackOff Back-off pulling image "registry.redhat.io/openshift4/ose-oauth-proxy:latest"
+// and marks it as a flake so we can see how often it's happening.
+//
+func testBackoffPullingRegistryRedhatImage(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
+
+	image_name := "\"registry.redhat.io/openshift4"
+	testName := "should be able to pull from registry.redhat.io"
+	success := &junitapi.JUnitTestCase{Name: testName}
+
+	for _, e := range events {
```

Comment:
we want similar threshold stuff like https://github.com/openshift/origin/pull/26855  Probably create a new struct that runs these test.